src/components/context-menu/ContextSubMenuItem.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+  import ChevronRightIcon from '@nasa-jpl/stellar/icons/chevron_right.svg?component';
+  import ContextMenu from './ContextMenu.svelte';
+  import ContextMenuItem from './ContextMenuItem.svelte';
+
+  export let text: string = '';
+  export let parentMenu: ContextMenu = null;
+
+  let contextMenu: ContextMenu;
+  let div: HTMLDivElement;
+
+  function show() {
+    if (div && contextMenu) {
+      contextMenu.showDirectly(0, 0, 0);
+      window.requestAnimationFrame(() => {
```

Comment:
Should we also [cancelAnimationFrame](https://developer.mozilla.org/en-US/docs/Web/API/Window/cancelAnimationFrame) on hide?

Reply:
Suppose so? I guess a user would have to trigger a show immediately followed by a hide within a single animation frame for there to be an issue? Guess maybe an automated test might run into this?

Reply:
Yeah just good to cleanup after ourselves with these things I think, even if rare

Reply:
Done.